api/src/main/resources/workbench-api.yaml

```diff
@@ -2249,29 +2268,6 @@ paths:
           description: Successful copy
           schema:
             "$ref": "#/definitions/ConceptSet"
-  "/v1/workspaces/{workspaceNamespace}/{workspaceId}/data-set/generateCode/{kernelType}":
```

Comment:
I thought about this when I was removing it but decided the payoff for just deleting it now and not having to tidy up later outweighed the benefit of keeping it around. From what I understand about our roll back process, if we discover a need to roll back, we would roll back both the API version and the UI version since that's the last stable release. Keeping this method would allow us to roll back just the UI version but I didn't think we would actually exercise that ability.

Reply:
Discussed further in Slack, recap: the main reason this matters is that anyone who is using the system when this kind of breaking change gets pushed will be in a bad state, and their export requests will fail until they reload the page.